types/lodash/index.d.ts

```diff
@@ -3918,42 +3918,42 @@ declare namespace _ {
         /**
          * @see _.zipWith
          */
-        zipWith<T, TResult>(
+        zipWith<T, U, TResult>(
             arrays: List<T> | null | undefined,
-            arrays2: List<T> | null | undefined,
-            iteratee: (value1: T, value2: T) => TResult
+            arrays2: List<U> | null | undefined,
+            iteratee: (value1: T, value2: U) => TResult
         ): TResult[];
 
         /**
          * @see _.zipWith
          */
-        zipWith<T, TResult>(
+        zipWith<T, U, V, TResult>(
             arrays: List<T> | null | undefined,
-            arrays2: List<T> | null | undefined,
-            arrays3: List<T> | null | undefined,
-            iteratee: (value1: T, value2: T, value3: T) => TResult
+            arrays2: List<U> | null | undefined,
+            arrays3: List<V> | null | undefined,
+            iteratee: (value1: T, value2: U, value3: V) => TResult
         ): TResult[];
 
         /**
          * @see _.zipWith
          */
-        zipWith<T, TResult>(
+        zipWith<T, U, V, W, TResult>(
             arrays: List<T> | null | undefined,
-            arrays2: List<T> | null | undefined,
-            arrays3: List<T> | null | undefined,
-            arrays4: List<T> | null | undefined,
-            iteratee: (value1: T, value2: T, value3: T, value4: T) => TResult
+            arrays2: List<U> | null | undefined,
+            arrays3: List<V> | null | undefined,
+            arrays4: List<W> | null | undefined,
+            iteratee: (value1: T, value2: U, value3: V, value4: W) => TResult
         ): TResult[];
 
         /**
          * @see _.zipWith
          */
-        zipWith<T, TResult>(
+        zipWith<T, U, V, W, X, TResult>(
             arrays: List<T> | null | undefined,
-            arrays2: List<T> | null | undefined,
-            arrays3: List<T> | null | undefined,
-            arrays4: List<T> | null | undefined,
-            arrays5: List<T> | null | undefined,
+            arrays2: List<U> | null | undefined,
+            arrays3: List<V> | null | undefined,
+            arrays4: List<W> | null | undefined,
+            arrays5: List<X> | null | undefined,
```

Comment:
This isn't quite right because `U`, `V`, `X`, and `W` are only used once each, and they're not used in the iteratee.

There's a PR that should be merged soon (#22126) that will change this, so you might want to wait for that to be merged before you make any changes.